Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java

```diff
@@ -82,12 +82,12 @@ public ListViewModel(FilteredEventsModel eventsModel) {
     public List<CombinedEvent> getCombinedEvents(Interval timeRange, RootFilterState filterState) throws TskCoreException {
         List<TimelineEvent> events = eventManager.getEvents(timeRange, filterState.getActiveFilter());
         
+        ArrayList<CombinedEvent> combinedEvents = new ArrayList<>();
+        
         if (events == null || events.isEmpty()) {
-            return Collections.EMPTY_LIST;
+            return combinedEvents; //return an empty list;
```

Comment:
Please elaborate on the reason for making these changes.

Reply:
There was a warning about returning an untyped list.